### PR TITLE
[issue#112] Adds a multi-select drop-down for tech stack to the Project Form

### PR DESCRIPTION
**Pull request by sushmukund26**
Changes include:

* Added 'react-select' as a dependency to use the CreatableSelect for the drop-down

* Created the drop-down in AddProject.tsx

* Populated state.techState with temporary data to ensure that the drop-down is not empty

* Added styles to match the existing form

Next steps:

* Action needs to be wired up after the user selects the tech stack - the handleChange function and error handling need to be modified accordingly

demo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "react-modal": "3.11.2",
     "react-router-dom": "5.1.2",
     "react-scripts": "3.2.0",
+    "react-select": "^3.1.0",
```
Comment (nurmerey) [inline package.json:20]: Let's remove `^` to lock in the library version, and prevent future unexpected library upgrades.

File: src/pages/AddProject.tsx [hunk 2]
```diff
@@ -14,20 +15,26 @@ type FormState = {
...
-    // techStack: string,
+    techStack: string,
   },
   serverError: {
     message: string
   },
   cancelModalOpen: boolean
 }
 
+const techStackOptions = [{value: 'django', label: 'Django'},
```
Comment (nurmerey) [inline src/pages/AddProject.tsx:32]: Let's add a comment above that this information should be coming from `getTagsForProjectID` API once #118 is finished.

**Review by nurmerey — CHANGES_REQUESTED** [review]
@sushmukund26 Thank you for your contribution! LGTM so far, added a few requests before merging it in!

File: package.json (hunk 1 above)
Comment (sushmukund26) [reply package.json]: Good catch. I have removed it

File: src/pages/AddProject.tsx (hunk 2 above)
Comment (sushmukund26) [reply src/pages/AddProject.tsx:34]: Done!

**Review by nurmerey — APPROVED** [review]
🥇  LGTM! Thank you!

**Pull request merged**